pkg/web/provider.go

```diff
@@ -64,7 +65,8 @@ func (p *Provider) listenAndServe(handler http.Handler) error {
 		return http.ListenAndServeTLS(address, p.CertFile, p.KeyFile, handler)
 	}
 
-	log.Infof("certPathTLS or keyPathTLS not found, defaulting to HTTP")
+	log.Info("Certificate and certificate key were not found, defaulting to HTTP")
+	log.Info("Janus Admin API started")
```

Comment:
This log message exists only for HTTP, but not for HTTPS. Also, log message should have static string and context for message above (can not leave a comment there), something like `log.WithField("address", address).Info("Listening Admin API @")`.

Same applies to main web server.